frontend/test/metabase/scenarios/admin/people/people.cy.spec.js

```diff
@@ -25,13 +30,9 @@ describe("scenarios > admin > people", () => {
     it("should render (metabase-enterprise#210)", () => {
       cy.visit("/admin/people");
 
-      cy.log("Assert it loads People by default");
-      cy.get(".PageTitle").contains("People");
+      assertTableRowsCount(TOTAL_USERS);
 
-      cy.get(".ContentTable tbody tr")
-        .as("result-rows")
-        // Bobby Tables, No Collection Tableton, No Data Tableton, None Tableton, Robert Tableton
-        .should("have.length", TOTAL_USERS);
+      cy.findByText("8 people found");
```

Comment:
Why hard code this value?
If we change the default snapshot in the future (very likely), this will fail.

Reply:
Fixed all of these, thanks!